to-do-list.css

```diff
@@ -0,0 +1,61 @@
+body {
+	background-color: #74AFAD;
+	text-align: center;
+	padding-left: 230px;
+	font-family: comic sans ms, cursive, sans-serif;
+
+}
+
+#page {
+	background-image: url('post-it-note.png');
+	background-repeat: no-repeat;
+	background-size: 830px;
+	display: block;
+	background-position: top center;
+	position: fixed;
+	width: 820px;
+	height: 700px;
+	overflow: scroll;
+	text-align: center;
+	padding-top: 35px;
+
+}
+
+#list-tasks {
+	width: 575px;
+	height: 290px;
+	overflow: scroll;
+	text-align: center;
```

Comment:
Here, you can set ```overflow: auto``` so that the scroll panes only appear if your list is longer than the div in which it's contained.